src/eip7594/eip7594.c

```diff
@@ -454,6 +454,63 @@ static C_KZG_RET compute_r_powers_for_verify_cell_kzg_proof_batch(
     return ret;
 }
 
+/**
+ * Compute the sum of the commitments weighted by the powers of r.
+ *
+ * @param[out] sum_of_commitments_out   The resulting G1 sum of the commitments
+ * @param[in]  unique_commitments       Array of unique commitments
+ * @param[in]  commitment_indices       Indices mapping to unique commitments
+ * @param[in]  r_powers                 Array of powers of r used for weighting
+ * @param[in]  num_commitments          The number of unique commitments
+ * @param[in]  num_cells                The number of cells
+ */
+static C_KZG_RET compute_commitment_sum(
```

Comment:
These are weighted too, right? For consistency, maybe this should be:
```suggestion
static C_KZG_RET computed_weighted_sum_of_commitments(
```

Reply:
Addressed in https://github.com/ethereum/c-kzg-4844/commit/e68de38097d2db24e2b3b243e1635b84c814b2c4